horovod/tensorflow/xla_mpi_ops.cc

```diff
@@ -570,6 +604,5 @@ XLA_REGISTER_CUSTOM_CALL_TARGET(CallbackHVDAllreduceDone, "CUDA");
 } // namespace tensorflow
 } // namespace horovod
 
-#endif // TENSORFLOW_VERSION >= 2006000000
-#endif // HAVE_CUDA
 #endif // HAVE_GPU
+#endif // TENSORFLOW_VERSION >= 2006000000
```

Comment:
These two lines seem to be missing in the file now:
```
XLA_REGISTER_CUSTOM_CALL_TARGET(CallbackHVDAllreduce, "ROCM");
XLA_REGISTER_CUSTOM_CALL_TARGET(CallbackHVDAllreduceDone, "ROCM");
```
(Their counterparts with `"CUDA"` can still be found at lines 600 and 601.)

Probably you would still want to have a preprocessor branch for "CUDA" and "ROCM" here?

Reply:
Good catch. Fixed in the new commit. Thank you.